bot.py

```diff
@@ -358,12 +389,18 @@ async def play(self, message, command_content):
                 )
             return
 
+        if len(re.findall(self.playlist_regex, command_content)) > 0:
```

Comment:
```suggestion
        if re.findall(self.playlist_regex, command_content):
```

Reply:
I wasn't sure that truth value of a list was documented. Using your suggestion.

[Truth value of things with `__len__()` or `__bool__()`](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)